src/pep8ext_naming.py

```diff
@@ -14,10 +13,6 @@
 
 __version__ = '0.7.0'
 
-LOWERCASE_REGEX = re.compile(r'[_a-z][_a-z0-9]*$')
-UPPERCASE_REGEX = re.compile(r'[_A-Z][_A-Z0-9]*$')
```

Comment:
It's a shame that `\u` and `\U` were only added in 3.3, otherwise we could improve upon these.

Reply:
I'm not sure how you would use them match uppercase characters. (AFAIK, `\u` is a escape sequence for Unicode code-points, or are you referring to something else?)